From 20dd2dc8e6e7767575ec75fa549757fae2ad253d Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Thu, 9 May 2019 22:33:49 +0300
Subject: [PATCH 9/9] Add myself as PRU maintainer

binutils/ChangeLog:

2019-05-09  Dimitar Dimitrov  <dimitar@dinux.eu>

        * MAINTAINERS: Add myself as PRU port maintainer.

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 binutils/MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/binutils/MAINTAINERS b/binutils/MAINTAINERS
index 87b23bf875..e5200b10da 100644
--- a/binutils/MAINTAINERS
+++ b/binutils/MAINTAINERS
@@ -124,6 +124,7 @@ responsibility among the other maintainers.
   PPC		   Alan Modra <amodra@gmail.com>
   PPC		   Peter Bergner <bergner@vnet.ibm.com>
   PPC vector ext   Aldy Hernandez <aldyh@redhat.com>
+  PRU		   Dimitar Dimitrov <dimitar@dinux.eu>
   RISC-V	   Palmer Dabbelt <palmer@sifive.com>
   RISC-V	   Andrew Waterman <andrew@sifive.com>
   RISC-V	   Jim Wilson <jimw@sifive.com>
-- 
2.11.0

